### ISSUE TITLE
Module not exist after installing

### ISSUE THREAD

**Issue by rdyzakya** [thiviyanT/IntelliGraphs#1]
After installing the intelligraphs package, I can't import the package.

I have tried the same thing in google colab, but it turns out the same output. I already check the package list through `pip freeze` and it shows that the package actually already installed.

My python version is 3.9.12 and the pip version is 21.2.4

**Comment by thiviyanT-sony** [comment]
Hi there! Thanks for reporting this. I will look into this issue. Did you use the pip install command to install this?

**Comment by rdyzakya** [comment]
Yes, I use the pip install command, I also tried using the conda command

**Comment by thiviyanT-sony** [comment]
Thanks for the info! I will look into this and get back to you. 

I am busy with a deadline at the moment. Most likely you will hear back from me next week.

**Comment by thiviyanT-sony** [comment]
May I ask, what your use case for this library is? Just curious :)

**Comment by thiviyanT** [comment]
Sorry, I forgot to follow up. The imports are working correctly. I will close this issue. Feel free to re-open this issue, if there is an issue on your machine.
